src/NetAppFiles/NetAppFiles.Test/ScenarioTests/AccountTests.ps1

```diff
@@ -0,0 +1,107 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+function Test-AccountCrud
+{
+    $resourceGroup = "pws-sdk-tests-rg-1"
```

Comment:
I think there might have been a good reason for having to do this. However I will look into changing it. There is a new version of the PowerShell required very soon. I hope this can wait until then.